hack/Makefile

```diff
@@ -65,7 +65,11 @@ ifneq ($(OCI_REGISTRY), $(OCI_REGISTRY_VERSION))
 endif
 
 .PHONY: install-requirements
-install-requirements: $(deps) $(GOPATH)/bin/goimports mdref
+install-requirements: $(deps) $(GOPATH)/bin/goimports mkbin mdref
+
+.PHONY: mkbin
```

Comment:
I would expect that the `.PHONY` target is not required. If the target is already present, we don't need the execution of `mkbin`

Reply:
Fixed with https://github.com/open-component-model/ocm/pull/880/commits/186d7abc99adfbc37c92467f0f07b1e520df614c